stubs/Flask-SocketIO/flask_socketio/__init__.pyi

```diff
@@ -21,7 +22,32 @@ class _HandlerDecorator(Protocol):
 class _ExceptionHandlerDecorator(Protocol):
     def __call__(self, exception_handler: _ExceptionHandler[_R_co]) -> _ExceptionHandler[_R_co]: ...
 
+class _SocketIOServerOptions(TypedDict, total=False):
+    client_manager: Incomplete
+    logger: Logger | bool
+    json: Incomplete
+    async_handlers: bool
+    always_connect: bool
+
+class _EngineIOServerConfig(TypedDict, total=False):
+    async_mode: Literal["threading", "eventlet", "gevent", "gevent_uwsgi"]
+    ping_interval: int  # seconds
```

Comment:
This can also be a two-tuple "For advanced control, a two element tuple can be given, where the first number is the ping interval and the second is a grace period added by the server."

I also suspect it can be a float instead of an int but haven't verified.

Reply:
Yeah, I think it can be float based on the test cases.

https://github.com/miguelgrinberg/python-engineio/blob/c9f9a7298086db34addabbb541e6e21412e7a5ac/tests/async/test_socket.py#L22